src/zappy_gui_src/Core/Network/Unpack.hpp

```diff
@@ -30,8 +30,99 @@ std::map<std::string, player_orientation_e> mapOrientation{
     {"3", WEST},
 };
 
+class Player {
```

Comment:
move variables in `private` class part. They don't be used as `public`. If you need them, make setter and getter function for these variables.